projects/lf-documentation/package.json

```diff
@@ -16,9 +16,11 @@
     "@angular/material": "~12.1.4",
     "@angular/platform-browser": "~12.1.1",
     "@laserfiche/lf-js-utils": "3.0.5",
+    "@laserfiche/types-lf-ui-components": "12.0.0--preview-2333574870",
```

Comment:
see my other comment, could we make this "DUMMY_VERSION" and install the current version before running npm i